src/ccache.c

```diff
@@ -894,7 +913,7 @@ process_preprocessed_file(struct mdfour *hash, const char *path, bool pump)
 				free(cwd);
 			}
 			if (should_hash_inc_path) {
-				hash_string(hash, inc_path);
+				hash_buffer(hash, inc_path, strlen(inc_path));
```

Comment:
Why this change?

Reply:
There is no change for the binary buffer, but for the text buffer it inserts a newline between strings. Which means that there will be a lot of newlines inserted into the input, making it harder to read...